static/index.html

```diff
@@ -52,7 +52,7 @@
       </div>
     </div>
     <div class="heading">
-      <img src="/static/images/globe.svg" class="logo" alt="World driven logo" height="60" width="60">
+      <img src="/static/images/favicon-32x32.png" class="logo" alt="World driven logo" height="60" width="60">
```

Comment:
Well, don't use a low quality favicon as logo on the page, when you have an svg.